refactored secret script

Climbatize opened the pull request:
# Description
Duplicated changes from MiniApp repo secret script to this one

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

donnie-jp commented on RInAppMessaging/Classes/Views/Components/OptOutMessageView.swift:
```diff
@@ -4,8 +4,11 @@ internal class OptOutMessageView: UIView {
         static let checkboxSize: CGFloat = 12
         static let spacing: CGFloat = 5
+        // Temporary solution for 4" devices
+        static let overflowMargin: CGFloat = 8
```
Why is it only a temporary solution?

imaginaris replied:
Temporary until we introduce pixel perfect designs that properly anticipate all screen sizes. Hopefully...

Commit:
refactored secret script

imaginaris commented on configure-secrets.sh:
```diff
@@ -1,5 +1,29 @@
+   - param[n]: a list of secrets environment variables to put in secret config file
+
+  Example usage:
+    $0 MiniApp RMA_API_ENDPOINT RAS_PROJECT_SUBSCRIPTION_KEY RAS_PROJECT_IDENTIFIER
```
I think this example doesn't match InAppMessaging :)

Climbatize replied:
I don't see what's wrong 😚

imaginaris replied:
Shouldn't that be `$0 InAppMessaging RIAM_CONFIG_URL...` instead of `$0 MiniApp...`?

Climbatize replied:
I was a joke, sorry XD
Sometimes I should precise ^^

Climbatize replied:
done

imaginaris reviewed (changes requested):
Great improvement 👍 
Just one comment to address.

Commit:
changed example message

example message was showing an example for MiniApp project. It should have been a more generic example, but now I have to modify it I can just adapt it to the project

donnie-jp commented on configure-secrets.sh:
```diff
@@ -40,8 +48,36 @@ echo "//" >> $SECRETS_FILE
-echo "RIAM_APP_SUBSCRIPTION_KEY = $SUBSCRIPTION_KEY" >> $SECRETS_FILE
\ No newline at end of file
+if [ $success = false ] ; then
+	echo "Before building the project you must set missing environment variables. See project README for instructions."
```
@Climbatize 👍 
I think the README might need updated too.

imaginaris replied:
But there are already some instructions in README to set up env variables for Test app.
